client/app/services/on-error.js

```diff
@@ -13,8 +13,6 @@ export default Ember.Service.extend({
 
       if (originalHandler) {
         originalHandler(error);
-      } else {
-        throw error;
```

Comment:
@grantovich.
 @pdw207 and I were experiencing some issues with being unable to close the flash message error and found out it was breaking here. We were wondering if we could just remove it?